src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,44 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+	private static final int fullStack = 11;

```

Comment:
Constants should be written uppercase.
This constant is related to stackCapacity. Changing only one leads to an error (inconsistency).